website/docs/getting-started/setup/installation-guides/aws-ecs-with-fargate.md

```diff
@@ -0,0 +1,218 @@
+                  "ssmmessages:CreateDataChannel",
+                  "ssmmessages:OpenControlChannel",
+                  "ssmmessages:OpenDataChannel"
+              ],
+              "Resource": "*"
+          }
+        ]
+      }
+```
+
+    
+### 6: Provision an external MongoDB v5.0 instance
+
+Generate the MongoDB URI and ensure the credential used has the ClusterMonitor Role attached.
+
+:::caution
+   Please don't run the built-in embedded MongoDB with the EFS, this could cause the Appsmith instance to crash.
+:::
+
+## ECS overview
+
+![ECS OVERVIEW](/img/ecs-overview.png)
+
+## Deploy Appsmith (using Fargate):
+   
+1. [Create an ECS Cluster](#step-1-create-an-ecs-cluster)
+2. [Create Task and Container Definitions](#step-2-create-task-and-container-definitions)
+3. [Create and Run an ECS Service](#step-3-create-and-run-an-ecs-service)
+
+Switch to the old AWS console UI to follow the steps below.
+
+### Step 1: Create an ECS Cluster
+
+1. Navigate to Amazon ECS, choose clusters on the sidebar, and select `Create Cluster`.
+
+![Creating Cluster in AWS ECS](/img/Create-cluster-with-aws-ecs.png)
+
+2. Choose **Networking Only**, and select the next step.
+
+3. Enter your cluster name.
+
+4. Create a VPC by defining the CIDR block and subnets. You can skip this step if you already have a VPC.
+
+5. Enable CloudWatch Container Insights.
+
+6. Hit the **Create button**.
+
+### Step 2: Create task and container definitions
+
+Once the cluster is created, you need to create a task that runs on the cluster formed in [**Step 1**](#step-1-create-an-ecs-cluster).
+
+1. On the sidebar, choose Task Definitions and Create a new Task Definition.
+2. Choose Fargate as the launch type, and proceed to the next step.
+3. Enter the task definition name.
+4. Set the task role to None.
+5. Select the default Network mode.
+6. Set Linux as the Operating system family.
+7. Set the Task Execution Role to the one created in the prerequisite step.
+
+8. Select the default Task execution IAM role created in the Prerequisite step.
+
+9. Set the required task size for memory & CPU (Minimum requirement: 2vCPU and 4 GB Memory).
+
+10. Go to the **Volumes** section and add a new volume. Enter the Name as `appsmith_stack`, set the Volume type as **EFS**, and set the **File System ID** to the EFS filesystem created in the prerequisite step. Leave the remaining fields with the default values.
+
+    ![EFS Volume Configuration](/img/ecs-efs.png)
+
+11. Configure **Appsmith container**.
+
+    1. Click the **Add container** button.
+    2. Enter the container name, and set the Image to `appsmith/appsmith-ce`
+    3. Add port mappings for the ports **80->80,443->443**
+    4. Set the _Mount points Source volume_ to `appsmith_stack` and set the Container path to `/appsmith-stacks`.
+
+    ![Storage Setting](/img/storage_settings_aws_fargate.png)
+
+    5. You can configure the Environment Values for the Appsmith in the Environment Section.
+
+    :::caution 
```

Comment:
Avoid callouts one after the other. It will remove emphasis. Change the position of either one . If that is not a possibility, keep the more important one in a callout and put the other as normal text